src/pages/courses/create.vue

```diff
@@ -49,17 +49,11 @@ export default {
 					description: this.course.description,
 				}); */
 
-				this.$toast.open({
-					message: "Kurs erstellt",
-					type: "is-success",
-				});
+				this.$toasted.success("Kurs erstellt");
```

Comment:
```suggestion
				this.$toast.success("Kurs erstellt");
```
according to the docs you should use `$toast` instead of `$toasted`